commerce_coordinator/apps/titan/clients.py

```diff
@@ -172,6 +172,24 @@ def complete_order(self, order_uuid, edx_lms_user_id):
             },
         )
 
+    def get_active_order(self, edx_lms_user_id):
+        """
+        Request Titan for the user's current open order
+
+        Args:
+            edx_lms_user_id: the edx.org LMS user ID of the user receiving the order
+        """
+
+        logger.info(f'TitanAPIClient.get_active_order called using {locals()}.')
+        response = self._request(
+            request_method='GET',
+            resource_path=f'accounts/{edx_lms_user_id}/active_order'
+        )
+        order_type = response['data']['type'] if response['data']['type'] else ''
+        order_response = response['data']['attributes'] if response['data']['attributes'] else []
+        order_response['orderType'] = order_type
```

Comment:
That is the use of `orderType` here? is not returning `response['data']['attributes']` enough here?

Reply:
removed orderType, was unnecessary forgot to remove before